src/validation/validation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Validate
+  def check_type(*vars, type)
+    raise 'Wrong type of variables' unless vars.each { |var| var.is_a? type }
+  end
+
+  def check_not_empty(*vars)
+    raise('Some variable is empty') if vars.each.map(&:empty?).all?
+  end
+
+  def check_positive(var)
+    raise('The house number should be positive.') unless var.positive?
```

Comment:
In all these cases we raise the `RuntimeError`, can we add specific exceptions for each case?

Reply:
done